src/test/java/bio/terra/service/snapshot/flight/create/CreateSnapshotAddEmailsToSamGroupStepTest.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.service.snapshot.flight.create;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.SnapshotAccessRequestResponse;
+import bio.terra.service.auth.iam.IamRole;
+import bio.terra.service.auth.iam.IamService;
+import bio.terra.service.snapshot.flight.SnapshotWorkingMapKeys;
+import bio.terra.service.snapshotbuilder.SnapshotRequestDao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.StepResult;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@Tag(Unit.TAG)
+@ExtendWith(MockitoExtension.class)
+class CreateSnapshotAddEmailsToSamGroupStepTest {
+
+  @Mock private IamService iamService;
+  @Mock private SnapshotRequestDao snapshotRequestDao;
+  @Mock private FlightContext flightContext;
+  private static final String RESEARCHER_EMAIL = "researcher@gmail.com";
```

Comment:
If this is only used on one test, could it be initialized in that test rather than globally?